src/pages/common/SortFilterHeader/SortFilterHeader.tsx

```diff
@@ -162,6 +179,34 @@ export const SortFilterHeader = ({
           />
         </FieldContainer>
       </Flex>
+      {isResearchStore(currentStore) && (
+        <Flex sx={_inputStyle}>
+          <FieldContainer>
+            <Select
+              options={statusOptions}
+              placeholder="Filter by status"
+              value={
+                (currentStore as ResearchStore).selectedStatus
```

Comment:
```suggestion
                currentStore.selectedStatus
```
nit(non-blocking): since `isResearchStore` is a type guard, we don't need this cast

Reply:
this was left over from when I first started getting round the typescript errors related to the howto store not having the researchStatus property.  Will clear it up.